utils/lazy-pages-fuzzer/src/wasmer_backend.rs

```diff
@@ -34,6 +34,9 @@ use crate::{
 #[derive(Clone)]
 struct InstanceBundle {
     instance: Instance,
+    // NOTE: Due to the implementation of lazy pages, which need to access the Store to retrieve globals,
+    // we have to use a second mutable reference to the Store in the form of a raw pointer
+    // to use it within the lazy pages' signal handler context.
```

Comment:
It would be nice to describe why it considered to safe

Reply:
Added